### ISSUE TITLE
Zyn-Fusion (VST2) : Crash on renaming Macro Learn slot's title.

### ISSUE THREAD

**Issue by ZAQraven** [zynaddsubfx/zyn-fusion-issues#364]
My OS is Ubuntu 22.04 LTS, And I use Ardour 6.9.0 I use Zyn-Fusion-build, mode=release.
The crash has happens that enabling the keyboard then typing and then Ardour crash. Does the bug occur on Ardour or the Zyn-Fusion?

Meanwhile, LV2 looks fine.

**Comment by fundamental** [comment]
Do you see any output on the console if you run ardour from a terminal? I suspect this is a bug which would impact the standalone, VST, and LV2, though there is a small chance it only impacts one of the three.

**Comment by pgervais** [comment]
@ZAQraven Checking in. Is the crash still happening?

**Comment by ChrisDAT20** [comment]
Yes, unfortunately the issue is still happening. And Zyn-Fusion even crashes very often when typing a file name when saving an instrument. On both Ardour 8.2.0 and 8.4.0. Opened another issue for this crashes already.

**Comment by fundamental** [comment]
It's my understanding that this bug _may_ have been fixed. Have you verified this or #394 occur with a git checkout of the user interface? If so this is fairly high priority. If not, then this may simply be an issue that is another strong motivator for a new release.

**Comment by ChrisDAT20** [comment]
I downgraded to 5:3.0.5-1kxstudio7 (LV2 version), and the issues are gone. One issue which remains is that there is no cursor in the file name field.

**Comment by fundamental** [comment]
Fair enough, though that doesn't help in determining if this is an active issue or one that simply needs a release of 3.0.7 which is in git. Opening up the git version I do see a cursor in the name field, so it looks like that behavior at least is working as intended for the next release.

**Comment by ChrisDAT20** [comment]
I fetched the git version from 28.12.2023, and there those issues happen.
